setup-nightly.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
```

Comment:
Below, on line 21:

```python
packages=setuptools.find_packages('epispot/'),
```

`setuptools.find_packages` is called with an argument `'epispot/'`, which shouldn't be necessary. Additionally, it ends in a forward slash which is now unsupported. The normal setup.py file in epispot does not have this complication, however, we are not sure if that file works entirely either.

@Quantalabs can we remove this and still have the script work?

Reply:
Yes, you should use:
```python
setuptools.find_packages()
```

Reply:
Ok, committing that